tensorflow/compiler/xla/pjrt/BUILD

```diff
@@ -1,6 +1,5 @@
 load("//tensorflow/core/platform:rules_cc.bzl", "cc_library")
-load("//tensorflow:tensorflow.bzl", "tf_cc_test")
-load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda")
+load("//tensorflow:tensorflow.bzl", "if_nccl", "tf_cc_test")
```

Comment:
I'm not sure (I can't find the error from copybara), but I'm guessing this line is the problem. As part of our internal CI these imports get rewritten, and I think the rewrite expects only one thing is loaded per load line. Can you try replacing this with:
```
load("//tensorflow:tensorflow.bzl", "if_nccl")
load("//tensorflow:tensorflow.bzl", "tf_cc_test")
```

?

If that doesn't work, I'll make the same change myself on your behalf to get you unblocked.

Reply:
Hmm, without changing anything, copybara passed. So should I edit as suggested or just wait?

Reply:
I think that change is still needed; it won't hurt either way, so I suggest making it and then hopefully the merge will go through.